appveyor.yml

```diff
@@ -92,16 +92,20 @@ for:
         cd %APPVEYOR_BUILD_FOLDER%
 
     build_script:
-    - cmd : >-
+    - ps : >-
         cd code\mobile\android\PhoneVR
         
-        gradlew.bat assemble
+        IF ($env:APPVEYOR_PULL_REQUEST_NUMBER){
+            .\gradlew.bat assembleDebug
+        } else {
+            .\gradlew.bat assemble
+        }
```

Comment:
In this way you are building *only* **debug** version of app and verifying whether there are any compile time errors, you are not building **release** version of the app. Commits a883c1a4bf4fd8df7136884a75259cb9a47d9119 and 11042c98aa7eaccb3a2f6c81a6d76376f4cf1070 tries to solve the issue of PR builds failing by appveyor in a better way, by [building both release and debug builds as well signing apk](https://github.com/ShootingKing-AM/PhoneVR/commit/11042c98aa7eaccb3a2f6c81a6d76376f4cf1070#diff-180360612c6b8c4ed830919bbb4dd459).

Reply:
i dont see a point of generating a release build for PR.

Reply:
Its not whether its debug/release, when you change the configuration of the compiler, it is possible that certain new errors will be generated with the same code for which debug build has successfully passed. And this project is distributing Release version, checks should to as close as possible to the distribution configuration.